lms/static/js/student_account/views/LoginView.js

```diff
@@ -102,7 +103,11 @@
             },
 
             saveError: function( error ) {
-                this.errors = ['<li>' + error.responseText + '</li>'];
+                if(error.status === 500){
+                    this.errors = ['<li>' + gettext("Something went wrong.") + '</li>'];
```

Comment:
@marcotuts @srpearce Can you please share what error message should we display on this error?